### DISCUSSION TITLE
Error in networks.create_model() when Using PoreSpy's AI-Based Diffusive Conductance Prediction

### DISCUSSION THREAD

**Discussion by DOMFIRMUS** [PMEAL/porespy#992] [Q&A] [answered]
Link: https://porespy.org/examples/networks/tutorials/using_diffusive_size_factor_AI_with_snow.html

Hi everyone,

I am trying to implement the AI-based diffusive conductance prediction in PoreSpy using the networks.create_model() function. However, I encountered an error when compiling the model. Below is a summary of the issue:

When executing the following code:

```
model = ps.networks.create_model()
model.load_weights(path_weights)
```

I get the following error:

`ValueError: Argument(s) not recognized: {'lr': 0.0001}`

It seems that the learning rate (lr) argument in the Adam optimizer is not being recognized, which causes the error when the model is compiled. The model is expected to use the Adam optimizer with a learning rate of 1e-4, but the current implementation throws this error.

Has anyone encountered a similar issue with PoreSpy or TensorFlow? Any suggestions on how to resolve this? I have already checked that the necessary dependencies (TensorFlow, Scikit-learn) are installed and up-to-date. Your help would be much appreciated!

Thanks in advance!

**Comment by ma-sadeghi** [comment, marked as answer]
Try installing Tensorflow v1:

```shell
pip install tensorflow-cpu==1.15.0
```

**Reply by DOMFIRMUS** [reply]
Thanks!

**Reply by ma-sadeghi** [reply]
Did it fix the issue?

**Reply by DOMFIRMUS** [reply]
Yes :)
